### PR TITLE
ci: update test configuration

### PR DESCRIPTION
**Pull request by johnhwhite**
- [ ] Fixes `Warning: ▲ [WARNING] Comments in CSS use "/* ... */" instead of "//" [js-comment-in-css]` errors by setting the `inlineStyleLanguage` for tests.
- [ ] Moves builder-specific configuration from CLI parameters to project configuration, so e.g., Jest tests don't get `--browsers=` parameter.
- [ ] Create `npx nx g @skyux-sdk/tools:configure-test-ci` for updating test configurations.

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/1870/storybook/)

Component Storybooks:

- [a11y-storybook](https://blackbaud.github.io/skyux-pr-preview/1870/storybooks/a11y-storybook/)
- [action-bars-storybook](https://blackbaud.github.io/skyux-pr-preview/1870/storybooks/action-bars-storybook/)
- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/1870/storybooks/ag-grid-storybook/)
- [angular-tree-component-storybook](https://blackbaud.github.io/skyux-pr-preview/1870/storybooks/angular-tree-component-storybook/)
- [avatar-storybook](https://blackbaud.github.io/skyux-pr-preview/1870/storybooks/avatar-storybook/)
- [colorpicker-storybook](https://blackbaud.github.io/skyux-pr-preview/1870/storybooks/colorpicker-storybook/)
- [core-storybook](https://blackbaud.github.io/skyux-pr-preview/1870/storybooks/core-storybook/)
- [data-manager-storybook](https://blackbaud.github.io/skyux-pr-preview/1870/storybooks/data-manager-storybook/)
- [datetime-storybook](https://blackbaud.github.io/skyux-pr-preview/1870/storybooks/datetime-storybook/)
- [errors-storybook](https://blackbaud.github.io/skyux-pr-preview/1870/storybooks/errors-storybook/)
- [flyout-storybook](https://blackbaud.github.io/skyux-pr-preview/1870/storybooks/flyout-storybook/)
- [forms-storybook](https://blackbaud.github.io/skyux-pr-preview/1870/storybooks/forms-storybook/)
- [indicators-storybook](https://blackbaud.github.io/skyux-pr-preview/1870/storybooks/indicators-storybook/)
- [inline-form-storybook](https://blackbaud.github.io/skyux-pr-preview/1870/storybooks/inline-form-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/1870/storybooks/layout-storybook/)
- [lists-storybook](https://blackbaud.github.io/skyux-pr-preview/1870/storybooks/lists-storybook/)
- [lookup-storybook](https://blackbaud.github.io/skyux-pr-preview/1870/storybooks/lookup-storybook/)
- [modals-storybook](https://blackbaud.github.io/skyux-pr-preview/1870/storybooks/modals-storybook/)
- [navbar-storybook](https://blackbaud.github.io/skyux-pr-preview/1870/storybooks/navbar-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/1870/storybooks/pages-storybook/)
- [phone-field-storybook](https://blackbaud.github.io/skyux-pr-preview/1870/storybooks/phone-field-storybook/)
- [popovers-storybook](https://blackbaud.github.io/skyux-pr-preview/1870/storybooks/popovers-storybook/)
- [progress-indicator-storybook](https://blackbaud.github.io/skyux-pr-preview/1870/storybooks/progress-indicator-storybook/)
- [split-view-storybook](https://blackbaud.github.io/skyux-pr-preview/1870/storybooks/split-view-storybook/)
- [storybook](https://blackbaud.github.io/skyux-pr-preview/1870/storybooks/storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/1870/storybooks/tabs-storybook/)
- [text-editor-storybook](https://blackbaud.github.io/skyux-pr-preview/1870/storybooks/text-editor-storybook/)
- [theme-storybook](https://blackbaud.github.io/skyux-pr-preview/1870/storybooks/theme-storybook/)
- [tiles-storybook](https://blackbaud.github.io/skyux-pr-preview/1870/storybooks/tiles-storybook/)
- [toast-storybook](https://blackbaud.github.io/skyux-pr-preview/1870/storybooks/toast-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/1870/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/1870/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/1870/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/1870/playground/)

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -244,8 +244,8 @@ jobs:
           key: ${{ runner.os }}-node-${{ needs.install-deps.outputs.node-version }}-modules-${{ hashFiles('package-lock.json') }}
       - name: Code coverage
         run: |
-          npx nx affected --target=test --browsers=ChromeHeadlessNoSandbox --code-coverage --parallel=3 --progress=false --source-map --watch=false
-          npx nx affected --target=posttest --browsers=ChromeHeadlessNoSandbox --parallel=3 --watch=false
+          npx nx affected --target=test --parallel=3 -c ci
+          npx nx affected --target=posttest --parallel=3 -c ci
```
Comment (Blackbaud-SteveBrush) [inline .github/workflows/ci.yml]: For the sake of longterm readability (and not having to guess what `-c` stands for), let's use `--configuration ci`.

File: apps/code-examples/project.json [hunk 2]
```diff
@@ -102,7 +102,20 @@
           "libs/components/theme/src/lib/styles/themes/modern/styles.scss"
         ],
         "scripts": [],
-        "assets": []
+        "assets": [],
+        "inlineStyleLanguage": "scss",
+        "stylePreprocessorOptions": {
```
Comment (Blackbaud-SteveBrush) [inline apps/code-examples/project.json:107]: Are these passed to `ng-packagr`? Do we need to include the `inlineStyleLanguage` and `includePaths` both in the `project.json` and the `ng-package.json`? Could we opt to only including it in one place?
See: https://github.com/blackbaud/skyux/blob/main/libs/components/a11y/ng-package.json

Comment (johnhwhite) [reply apps/code-examples/project.json:107]: These are passed to the [karma builder](https://github.com/angular/angular-cli/blob/main/packages/angular_devkit/build_angular/src/builders/karma/schema.json#L117), it receives its options differently than the `ng-packagr` builder, and I don't see a way to combine the two set of options.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (johnhwhite) [reply .github/workflows/ci.yml]: Good call. Updated.

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**